### ISSUE TITLE
Bring HAFAS to HACS

### ISSUE THREAD

**Issue by akloeckner** [akloeckner/hacs-hafas#1]
Hey @kilimnik,

I felt so free to take your work from https://github.com/home-assistant/core/pull/78839 and cast it into a very preliminary HACS repository. You should be able to add this custom repository (https://github.com/akloeckner/hacs-hafas) to HACS (following [this guide](https://hacs.xyz/docs/faq/custom_repositories/)) and create a working Deutsche Bahn sensor (yeay):
![grafik](https://user-images.githubusercontent.com/7125061/214447970-61fcf4fa-1f7b-4573-af18-b638c6a85bc5.png)

However, I found a severe issue, that I wouldn't want to discuss in the HA PR. I noticed that my local Kölner Verkehrsbetriebe (KVB) was not available as a profile. So, I investigated how to add it as a profile. And it seems, that requires changing the [pyhafas](https://github.com/FahrplanDatenGarten/pyhafas) library. Which has several open PRs, partly dating way back to 2021. So, I got the feeling that this library might not be really maintained anymore. Even if it is maintained, every new company would require a code change. Do you think, that will be usable by the regular HA user?

Then again, if you think, that is doable, we could be close to having a working HACS integration. :-) Maybe polish the readme, fix some minor bugs on the way and advertise. ;-)

**Comment by kilimnik** [comment]
Oh, wow. I will check this out. Thank you for the work

I saw that the library is not greatly maintained. Unfortunately it's the best/only one out there. It would be possible to write the integration to use all the profiles, the library supports. But I am not sure, if that's needed. Most Trains and Busses in Germany are also queryable via the DB profile. I haven't seen any that are not.

**Comment by akloeckner** [comment]
> Most Trains and Busses in Germany are also queryable via the DB profile.

Yes, that's a good argument. It will be sufficient for most users to just use the DB interface. In my case, I have noticed that DB is not as up-to-date as the local app is. But that might be an edge case. Which I should be able to fix with a single new profile.

> I saw that the library is not greatly maintained.

Maybe, that will change once the user base becomes bigger through your integration. ;-)

**Comment by akloeckner** [comment]
That's maybe something @kilimnik can help with. I remember to have seen a "non-speaking" error message like that, too. Then, I spelled the stations differently and it worked. So, I guess the "unknown" refers to the station name?

**Comment by akloeckner** [comment]
@kilimnik, I have invited you as an collaborator. But if you feel this project should live in your account, let's migrate it. I'm fine with either location.

**Comment by akloeckner** [comment]
I'll close the issue to tidy up a bit...
